tests/test_great_expectations.py

```diff
@@ -312,12 +322,14 @@ def test_stats_mixed_expectations():
 
 
 class TestIO(unittest.TestCase):
+    @pytest.mark.filesystem
```

Comment:
```suggestion

@pytest.mark.filesystem
class TestIO(unittest.TestCase):
```

You can mark the `TestClass` itself to cause all of its methods to become marked as well.

https://docs.pytest.org/en/7.1.x/example/markers.html#marking-whole-classes-or-modules

```python
import pytest


@pytest.mark.webtest
class TestClass:
    def test_startup(self):
        pass

    def test_startup_and_more(self):
        pass
```

Reply:
Thank you for this @Kilo59 a few tests in this TestClass have a `pyarrow` mark instead of a `filesystem` mark.

Reply:
I'll keep it in mind for upcoming test classes.